### ISSUE TITLE
Docs addon propType description broken when using CRA and emotion's jsx pragma

### ISSUE THREAD

**Issue by jscheel** [storybookjs/storybook#9150]
When using the jsx pragma for the css prop from emotion, storybook's docs addon does not render propType descriptions.

Here is a repo that reproduces the issue:
https://github.com/jscheel/storybook-cra-emotion-jsx-pragma

The repo provides a working and broken example. In the working example (`NormalExample`), the description column of the prop table is generated properly. In the broken example (`EmotionExample`), the description column is blank.

If I kill the create-react-app config (easiest way is to just `npm remove react-scripts`), everything works as expected in both examples.

**System:**
```
Environment Info:

  System:
    OS: macOS Sierra 10.12.5
    CPU: (8) x64 Intel(R) Core(TM) i7-4850HQ CPU @ 2.30GHz
  Binaries:
    Node: 12.13.0 - /var/folders/27/fqmq58ks6kzbkmmgg7whv4280000gn/T/fnm-shell-2901125/bin/node
    Yarn: 1.13.0 - ~/.yarn/bin/yarn
    npm: 6.12.0 - /var/folders/27/fqmq58ks6kzbkmmgg7whv4280000gn/T/fnm-shell-2901125/bin/npm
  Browsers:
    Chrome: 79.0.3945.79
    Firefox: 70.0.1
    Safari: 10.1.1
  npmPackages:
    @storybook/addon-actions: ^5.2.8 => 5.2.8
    @storybook/addon-docs: ^5.2.8 => 5.2.8
    @storybook/addon-links: ^5.2.8 => 5.2.8
    @storybook/addons: ^5.2.8 => 5.2.8
    @storybook/react: ^5.2.8 => 5.2.8
```

Normal, Working Example:
![normal](https://user-images.githubusercontent.com/211478/70849474-7e12eb80-1e44-11ea-8253-11ad4c6618f7.png)

Broken, Emotion Example:
![emotion](https://user-images.githubusercontent.com/211478/70849475-84a16300-1e44-11ea-9819-dbc310be1e9f.png)

**Comment by shilman** [comment]
Thanks for the repro, although it doesn't work for me at all. Did you test it?

It complained about not having `@storybook/addon-knobs` installed, so I installed it.

After that, I tried to install the `preset-create-react-app` standalone preset, which I expect would fix the problem:

https://www.npmjs.com/package/@storybook/preset-create-react-app

However, it gave me a babel error. @mrmckeb can you take a look?

**Comment by jscheel** [comment]
I’ll attach some screenshots in a bit

**Comment by jscheel** [comment]
@shilman ok, I've updated the repo with the missing package. I was deleting some unused packages before committing and accidentally took out `@storybook/addon-knobs`.

I've added screenshots of each example in my repo and here so you can see the problem.

I also tried `preset-create-react-app` and had issues as well.

**Comment by mrmckeb** [comment]
I'm unable to test this as I'm hitting this issue... https://github.com/inspect-js/is-set/issues/3.

You also should be able to skip the `babel.config.js` file, as we load the required config from CRA.

**Comment by hckhanh** [comment]
I get the same issue when I try to upgrade Storybook. My props are not displayed properly anymore. It does not show any props.

**Comment by shilman** [comment]
i think this is the underlying issue: https://github.com/storybookjs/presets/issues/78

will try to work through that, and then look into the emotion issue after it's resolved. thanks for your patience!

**Comment by shilman** [comment]
Fixed in `@storybook/preset-create-react-app@1.5.0`!! Please upgrade and try it out! Thanks @mrmckeb for the fix 🙌 

Then we can get back to the original issue about jsx pragma 😉

**Comment by mrmckeb** [comment]
Can this issue be closed? It appears to have been resolved.
